### PR TITLE
Decide team name and come up with questions/highlights

### PR DESCRIPTION
**Pull request by SBucharelli**

### CODE REVIEW

File: QUESTIONS.md [hunk 1]
```diff
@@ -3,3 +3,17 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+- How does hosting our website work? Good sites to host from/ buy domains?
```
Comment (crkingsley) [inline QUESTIONS.md]: +1, I wonder about this too!

File: QUESTIONS.md [hunk 2]
```diff
@@ -3,3 +3,17 @@
...
+
+- Stephanie can't wrap her head around React. She's having an allergic Reaction!
```
Comment (crkingsley) [inline QUESTIONS.md]: +1

**Review by crkingsley — APPROVED** [review]
LGTM!
You have some REALLY great questions!! I hope we get an answer to them!

File: TEAMS.md [hunk 3]
```diff
@@ -1 +1,3 @@
 # Week 13 Teams and Rosters
+
+- The (Vegan) Baconators!
```
Comment (Mvrcusj) [inline TEAMS.md]: Digging the team name

File: QUESTIONS.md [hunk 4]
```diff
@@ -3,3 +3,17 @@
...
+
+- Want to know more about middleware, is this where security is provided? Why does your app need a hat (helmet)?
```
Comment (hollisjamison) [inline QUESTIONS.md]: Small version: middleware is an express function that every request passes through before you route it to a specific place. Helmet scans that request and looks for certain kinds of bad stuff to protect you. So yes middleware CAN protect you but not all middleware is for this purpose.

File: TEAMS.md (hunk 3 above)
Comment (Deva0770) [reply TEAMS.md]: Sounds ok..!!

**Pull request merged**